src/RobotInterface/tests/DummyImplementation/DummySensorBridgeTest.cpp

```diff
@@ -0,0 +1,55 @@
+/**
+ * @file DummySensorBridgeTest.cpp
+ * @authors Prashanth Ramadoss
+ * @copyright 2020 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the GNU Lesser General Public License v2.1 or any later version.
+ */
+
+
+// Catch2
+#include <catch2/catch.hpp>
+#include <BipedalLocomotion/ParametersHandler/IParametersHandler.h>
+#include <BipedalLocomotion/ParametersHandler/StdImplementation.h>
+
+#include "./DummySensorBridge.h"
```

Comment:
I guess you can remove ``./``. When using ``"``, it already looks in the current folder.